docs/coding-guidelines/project-guidelines.md

```diff
@@ -29,7 +29,7 @@ Below is a list of all the various options we pivot the project builds on:
 The following are the properties associated with each build pivot
 
 - `$(BuildTargetFramework) -> netstandard2.1 | netcoreapp5.0 | net472`
-- `$(OSGroup) -> Windows | Linux | OSX | FreeBSD | [defaults to running OS when empty]`
+- `$(BuildOS) -> Windows | Linux | OSX | FreeBSD | [defaults to running OS when empty]`
 - `$(Configuration) -> Release | [defaults to Debug when empty]`
 - `$(ArchGroup) - x86 | x64 | arm | arm64 | [defaults to x64 when empty]`
 - `$(RuntimeOS) - win7 | osx10.10 | ubuntu.14.04 | [any other RID OS+version] | [defaults to running OS when empty]` See [RIDs](https://github.com/dotnet/runtime/tree/master/src/libraries/pkg/Microsoft.NETCore.Platforms) for more info.
```

Comment:
What is RuntimeOS ?

Reply:
RuntimeOS refers to  the rids.

Reply:
It seems confusing that the `*OS` property is actual OS name, and a different `*OS` property is OS RID.

Reply:
we are working towards removing these *OS properties. Currently there are a lot of dangling properties among the three subsets and each of them uses its own kind of naming. I am trying to unify them and hopefully remove as much as possible